src/config_web.py

```diff
@@ -11,6 +11,7 @@
 ES_HOST = 'localhost:9200'
 # elasticsearch index name
 ES_INDEX = 'mygeneset_current*'
+ES_CURATED_INDEX = 'mygeneset_current'
 ES_USER_INDEX = 'mygeneset_current_user_genesets'
```

Comment:
Since you defined both `ES_CURATED_INDEX` and `ES_USER_INDEX`, `ES_INDEX` can now be set as `[ES_CURATED_INDEX, ES_USER_INDEX]`. Double check if it impacts the `biothing_type` value in the metadata.

Reply:
Changed it and the `biothing_type` value kept the same. As we can see below:

```
"biothing_type": [
  "geneset"
],
```